tests/system/action/mediafile/test_upload.py

```diff
@@ -0,0 +1,94 @@
+import base64
+from unittest.mock import patch
+
+from tests.system.action.base import BaseActionTestCase
+
+
+class MediafileUploadActionTest(BaseActionTestCase):
+    @patch("openslides_backend.action.mediafile.upload.Mediaservice")
+    def test_create(self, mock) -> None:
+        self.create_model("meeting/110", {"name": "name_DsJFXoot"})
+        response = self.client.post(
+            "/",
+            json=[
+                {
+                    "action": "mediafile.upload",
+                    "data": [
+                        {
+                            "title": "title_xXRGTLAJ",
+                            "meeting_id": 110,
+                            "filename": "fn_jumbo.txt",
+                            "file": base64.b64encode(b"testtesttest").decode(),
+                        }
+                    ],
+                }
+            ],
+        )
+        self.assert_status_code(response, 200)
+        mediafile = self.get_model("mediafile/1")
+        assert mediafile.get("title") == "title_xXRGTLAJ"
+        assert mediafile.get("meeting_id") == 110
+        assert mediafile.get("filename") == "fn_jumbo.txt"
+        assert mediafile.get("file") is None
```

Comment:
Please assert that `upload` was called. The mocked method should have a `assertCalledWith()` function.